src/hotspot/share/gc/shared/cardTable.hpp

```diff
@@ -40,6 +40,7 @@ class CardTable: public CHeapObj<mtGC> {
   // This code can typically be found by searching for the byte_map_base() method.
   STATIC_ASSERT(sizeof(CardValue) == 1);
 
+  using CardWord = uintptr_t;
```

Comment:
Not really convinced this is a good name, but I do not have a good alternative either right now.